src/components/TeamMember.js

```diff
@@ -0,0 +1,33 @@
+import {
+  Box,
+  Card,
+  CardContent,
+  Typography,
+} from "@material-ui/core/";
+import useMyStyles from "../styles/MyStyles";
+import AccountCircleSharpIcon from "@material-ui/icons/AccountCircleSharp";
+
+const TeamMember = (props) => {
+  const classes = useMyStyles();
+
+  return (
+    <Card
+      key={props.id}
+      className={classes.card__TeamMember}
+      variant="outlined"
+    >
+      <CardContent className={classes.cardContent__TeamMember}>
```

Comment:
would be nice to read about css class naming, for example BEM
it's not the best practice to add in class name part of html tag, now it's called `card_teamMember`, but it will be confusing if in some time you will use for example `<Chip>` because layout changed
I suggest to keep closer to functionality of element (wrapper, header, body) on TeamMember example it could be (BEM style) `team-member__wrapper` `team-member__name` `team-member__image`

Reply:
I changed names of styles as suggested. Because I'm using _makeStyles_ I'm not able to use hyphens, so I did like "TeamMember__mainWrapper" to at first point what type of component I'm styling and then more generic name for function of the styled element.